rskj-core/src/test/java/co/rsk/config/BridgeConstantsTest.java

```diff
@@ -0,0 +1,69 @@
+package co.rsk.config;
+
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class BridgeConstantsTest {
+
+    private static Stream<Arguments> generator() {
+        return Stream.of(
+            Arguments.of(BridgeMainNetConstants.getInstance(), false),
+            Arguments.of(BridgeTestNetConstants.getInstance(), true),
+            Arguments.of(BridgeRegTestConstants.getInstance(), true)
+        );
+    }
+
+    @ParameterizedTest()
+    @MethodSource("generator")
+    void test_getFundsMigrationAgeSinceActivationEnd(BridgeConstants bridgeConstants, boolean hasSameValueForBothMigrationAges) {
```

Comment:
Is it possible to also parameterize RSKIP activations? That would be cool

Reply:
It may be possible, but it will also imply parameterizing the assertions or adding if's,  as the assertions depend on the RSKIP activated. I believe it will make tests complex.